Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/controllers/Settings.Controller.js

```diff
@@ -106,8 +106,12 @@
                 //Don't need anything from response.data back
 
                 //Display Success message
-                notificationsService.success("Success profile details have been saved");
+                notificationsService.success(localizationService.localize("analytics_profileDetailsSaved"));
             });
+
+            //Sync ('refresh') the tree!
+            //This should work, but somehow it seems to fails sometimes when you click "save" and receive an error in a dialog on the right.
+            //navigationService.syncTree({ tree: 'analyticsTree', path: [-1, -1], forceReload: false, activate: true });
```

Comment:
This should work to refresh the trees from root after save. However on some clicks I get this error and the trees disappear, so I have to refresh the page/reload the trees with the the action refresh menu to see the trees again.

![2015-02-20_23-57-31](https://cloud.githubusercontent.com/assets/2919859/6314126/2c38911c-b9d0-11e4-80e9-855f36639191.jpg)

There have been some issues about this:
http://issues.umbraco.org/issue/U4-5361
http://issues.umbraco.org/issue/U4-6215

Maybe this is fixed in Umbraco 7.2.2, so the test could be upgraded to 7.2.2. which have a lot of bug fixes.